css-positioning/absolute.css

```diff
@@ -0,0 +1,44 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  position: relative;
+
+  /* your code here */
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  bottom: 75%;
+  left: 75%;
```

Comment:
This is a _big_ movement. Instead of moving it all the way from the other side, you should try moving from `top` and `right`